editor/src/components/editor/actions/migrations/migrations.spec.ts

```diff
@@ -8,11 +8,4 @@ describe('Migrations', () => {
     const migrated = applyMigrations(emptyProject)
     expect(migrated.projectVersion).toEqual(CURRENT_PROJECT_VERSION)
   })
-
-  it('returns with the latest project version for a non-versioned project', () => {
```

Comment:
Hmmmm this test is definitely wrong, but this also means we have no test to ensure that the migrations actually run for an old project :/ Is there any chance you can grab an old model and hardcode that in here? I'm concerned that there's a chance this PR will destroy prior projects.

Reply:
The problem is that I'd need to get one from before all the migrations were implemented, which is a long time ago now.

Reply:
I did manage to find a project from a couple of months ago and that seems to work with the migration I added in this PR.